src/webapp/TourneyRent/src/app/pages/profile/profile.component.ts

```diff
@@ -56,6 +59,14 @@ export class ProfileComponent {
           status: this.tournamentService.getTournamentStatus(t)
         }));
       })
+
+    this.teamResource
+    .getUserTeams(this.authService.getAuthUserId())
+    .subscribe(teams => {
+      this.teams = teams.map((t: any) => ({
+        ...t
```

Comment:
you don't need to use map(), since you're not modifying anything https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map